conda_smithy/feedstock_tokens.py

```diff
@@ -175,6 +185,75 @@ def feedstock_token_exists(user, project, token_repo):
     return exists
 
 
+def is_valid_feedstock_token(user, project, feedstock_token, token_repo):
```

Comment:
How is this function supposed to be used?

Reply:
Ummmm, I thought that was obvious. You feed it the user/org, the name of the feedstock, the plaintext feedstock token, and the https address of the repo with the master list of tokens. It then returns `True` if the token is the correct one for the given feedstock and `False` otherwise.

Reply:
Do you have a suggestion for the docstring to make it easier to understand?

Reply:
I mean who is supposed to invoke this command?

Reply:
This command is used by the admin web services to verify the feedstock tokens it gets. It is here in smithy because you have to hash the tokens in exactly the same way you did when you registered them and so I figure it was easier to keep that code in sync by having it all in one spot.

Reply:
Sound good to me